Sources/MongoSwift/Operations/ListDatabasesOperation.swift

```diff
@@ -25,20 +25,35 @@ internal enum ListDatabasesResults {
     case names([String])
 }
 
+/// Options for "listDatabases" operations
+public struct ListDatabasesOptions {
+    /// Specifies whether to only return databases for which the user has privileges,
```

Comment:
I think my comment on this got lost before, but let's move the description of the potential error case out of here and into the individual method docstrings' error lists, something like:
"- An [XError] if the`authorizedDatabases` option was set to `false` but the user does not have the privilege to list all databases in the cluster."

(not sure what type of error that comes back as, maybe just a CommandError? worth checking)